src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java

```diff
@@ -16,12 +16,25 @@ public class HoppityEggsConfig {
     @FeatureToggle
     public boolean waypoints = true;
 
+    @Expose
+    @ConfigOption(name = "Waypoint Line", desc = "Draw a line from your eyes to the Hoppity's Hunt Waypoint.")
+    @ConfigEditorBoolean
+    public boolean drawLine = true;
```

Comment:
i feel this should be default off

Reply:
nvm will use whatever #1669 is

Reply:
Well there was a line already, so i had it default on. But i'm removing this from my pr since hanni already has a pr for the same thing (same implementation too lmao except i didn't add it to shared waypoints)